app/views/patients/show.html.erb

```diff
@@ -126,10 +130,14 @@
           <td><%= cc.last_name %></td>
           <td><%= cc.primary_telephone.present? && cc.primary_telephone.length > 10 ? cc.primary_telephone&.gsub('+1', '')&.gsub(/\D/, '')&.insert(3, '-')&.insert(7, '-') : '' %></td>
           <td><%= cc.email %></td>
-          <td><%= cc.contact_attempts %></td>
+          <td class="text-center"><%= cc.last_date_of_exposure&.to_date&.strftime('%m/%d/%Y') || 'Unknown' %></td>
+          <td class="text-center"><%= cc.assigned_user %></td>
+          <td><%= cc.notes %></td>
```

Comment:
I think we should limit the amount of text from the notes we actually display in the tables. On smaller screens it can be overwhelming at max character count. I would imagine we want to cut it off with an ellipsis at a certain point. Please discuss with PO team.
![Screen Shot 2021-01-28 at 10 12 32 AM](https://user-images.githubusercontent.com/11698457/106158229-8665dd00-6151-11eb-9af8-87de006431c5.png)

Reply:
funny you should mention this! i actually had this trimmed earlier to 40 characters plus `...` but decided to remove it. Will ask the PO team.

Reply:
The PO team said they wanted it trimmed to 500 char. Added in latest commit.

Reply:
Right now it looks like this when I enter a note with no spaces/line breaks. I think there needs to be wrapping implemented in this column as well. 
![Screen Shot 2021-01-29 at 10 13 28 AM](https://user-images.githubusercontent.com/11698457/106292487-deb4e180-621a-11eb-8d78-2599619b163e.png)

Reply:
Good catch! Never would've thought to test a string that long. Fixed!